src/main/java/ladder/domain/RandomLineSourceGenerator.java

```diff
@@ -0,0 +1,16 @@
+package ladder.domain;
+
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class RandomLineSourceGenerator implements LineSourceGenerator {
+
+
+    @Override
+    public LineSource generate() {
+        List<LineSource> values = new ArrayList<>(List.of(LineSource.values()));
```

Comment:
```suggestion
        List<LineSource> values = List.of(LineSource.values());
```

혹시 따로 ArrayList로 다시한번 포장하신 이유가 있었나요 ? 그렇다면 다시 알려주세요 ㅎㅎ